src/OpenDiffix.Core.Tests/QueryValidator.Tests.fs

```diff
@@ -2,33 +2,50 @@ module OpenDiffix.Core.QueryValidatorTests
 
 open Xunit
 
-let testTable: Table =
-  {
-    Name = "table"
-    Columns =
-      [
-        { Name = "str_col"; Type = StringType }
-        { Name = "int_col"; Type = IntegerType } // AID column
-        { Name = "float_col"; Type = RealType }
-        { Name = "bool_col"; Type = BooleanType }
-      ]
+let testTables =
+  [
+    {
+      Name = "anon_table"
```

Comment:
you might instead switch the AccessLevel between `Trusted` and `Direct`. Maybe that's more concise.

Reply:
I would prefer to control this from the query statement instead of needing to pass additional arguments to the helper test functions.